tests/SystemTests.cs

```diff
@@ -174,6 +174,7 @@ await testContext.AssertPullRequests(
     public async Task Given_Microsoft_Dependencies_Updates_Then_Opens_Major_PR_And_AutoMerges_Minor()
     {
         await using var testContext = await TestContext.CreateAsync(testOutputHelper);
+        testContext.UseRenovateFile("microsoft-automerge.json");
```

Comment:
SHould we do this test with and without the auto-merge renovate?

Reply:
The test with the default renovate could also add some Workleap dependencies to test both at the same time